python/ray/serve/utils.py

```diff
@@ -257,3 +265,19 @@ def parse_import_path(import_path: str):
         )
 
     return ".".join(nodes[:-1]), nodes[-1]
+
+
+class JavaActorHandleProxy:
+    """Wraps actor handle and translate snake_case to camelCase."""
```

Comment:
lol...

Reply:
is this necessary?

Reply:
sadly yes. otherwise our codebase will have even more branches https://github.com/ray-project/ray/issues/21474